deployment/deployment-caikit-embeddings.yaml

```diff
@@ -44,6 +38,8 @@ spec:
               value: /opt/app-root/src/.cache/huggingface/hub/
             - name: HUGGINGFACE_HUB_CACHE
               value: $(TRANSFORMERS_CACHE)
+            - name: MODELS_DIR
+              value: '["sentence-transformers_all-MiniLM-L6-v2","slate.rtvr271M"]'
```

Comment:
Eventually (e.g. in caikit-nlp) there would not be default models.  So I think we'd make this more dynamic based on the caikit config and fail before using a hard-coded default.  If we find that we're keeping a fallback default like this for a while, then it probably should be only open source examples.